pkg/network/common/egressip.go

```diff
@@ -290,6 +300,130 @@ func (eit *EgressIPTracker) UpdateHostSubnetEgress(hs *networkv1.HostSubnet) {
 	eit.syncEgressIPs()
 }
 
+func nodeIsReady(node *v1.Node) bool {
+	nodeReady := true
+	for _, cond := range node.Status.Conditions {
+		if cond.Type == v1.NodeReady {
+			if cond.Status == v1.ConditionFalse || cond.Status == v1.ConditionUnknown {
+				nodeReady = false
+			}
+		}
+	}
+	return nodeReady
```

Comment:
this function doesn't actually need `nodeReady`. You can just `return false` if you find a non-ready condition, and `return true` at the end if you don't

Reply:
I agree, I will change that.